test/tracing/mzcompose.py

```diff
@@ -26,12 +32,13 @@
 
 def workflow_default(c: Composition) -> None:
     """Tests the dynamic tracing setup on environmentd"""
-    c.workflow("with-otel")
-    c.workflow("without-otel")
-    c.workflow("clusterd")
+    for _, name in enumerate(c.workflows):
```

Comment:
Sorry, my example had a useless `enumerate`, this should work too, but not important enough for a new push: `for name in c.workflows:`